components/TopBarProfileMenu.js

```diff
@@ -38,12 +41,69 @@ const HideGlobalScroll = createGlobalStyle`
 `;
 
 const UserMenuLinkEntry = props => {
+  const { profileMenuLink } = props;
   return (
-    <ListItem mb="6px">
+    <ListItem mb="6px" py={profileMenuLink ? '2' : '0'}>
       <StyledLink as={Link} fontWeight="500" fontSize="14px" lineHeight="20px" color="black.800" {...props} />
     </ListItem>
   );
 };
+UserMenuLinkEntry.propTypes = {
+  profileMenuLink: PropTypes.bool,
+};
+
+const MyAccount = props => {
+  const { LoggedInUser, logout } = props;
+
+  return (
+    <Box as="ul" p={3} my={2}>
+      <Query
+        query={memberInvitationsCountQuery}
+        variables={{ memberCollectiveId: LoggedInUser.CollectiveId }}
+        fetchPolicy="network-only"
+      >
+        {({ data, loading }) =>
+          loading === false && data && data.memberInvitations && data.memberInvitations.length > 0 ? (
+            <UserMenuLinkEntry profileMenuLink href="/member-invitations">
+              <FormattedMessage
+                id="menu.pendingInvitations"
+                defaultMessage="Pending Invitations ({numberOfInvitations})"
+                values={{ numberOfInvitations: data.memberInvitations.length }}
+              />
+            </UserMenuLinkEntry>
+          ) : null
+        }
+      </Query>
+      <UserMenuLinkEntry profileMenuLink href={`/${LoggedInUser.collective.slug}/edit`}>
+        <FormattedMessage id="Settings" defaultMessage="Settings" />
+      </UserMenuLinkEntry>
+      <UserMenuLinkEntry profileMenuLink href={`/${LoggedInUser.username}/recurring-contributions`}>
+        <FormattedMessage id="menu.subscriptions" defaultMessage="Manage Contributions" />
+      </UserMenuLinkEntry>
+      <UserMenuLinkEntry profileMenuLink href={`/${LoggedInUser.username}/transactions`}>
+        <FormattedMessage id="menu.transactions" defaultMessage="Transactions" />
+      </UserMenuLinkEntry>
+      <UserMenuLinkEntry profileMenuLink href="/applications">
+        <FormattedMessage id="menu.applications" defaultMessage="Applications" />
+      </UserMenuLinkEntry>
+      <UserMenuLinkEntry profileMenuLink as="a" href="https://docs.opencollective.com">
+        <FormattedMessage id="menu.help" defaultMessage="Help" />
+      </UserMenuLinkEntry>
+      <UserMenuLinkEntry profileMenuLink as="a" data-cy="logout" onClick={logout}>
+        <Flex alignItems="center">
+          <P color="#7A0F2B" fontWeight="500" pr={2} whiteSpace="nowrap">
+            <FormattedMessage id="menu.logout" defaultMessage="Log out" />
+          </P>
+          <Exit size={13} color="#7A0F2B" />
```

Comment:
```suggestion
          <Exit size={13} color="red.800" />
```

Reply:
That won't work here since the icon comes from a third-party library that isn't wrapped by `styled-system`.

Reply:
@Betree : Ops... Thanks for pointing it out. 😉